### PR TITLE
OCPBUGS#18190: Added missing DNS records for vSphere IPI docs

### PR DESCRIPTION
**Pull request by dfitzmau**
[OCPBUGS-18190](https://issues.redhat.com/browse/OCPBUGS-18190)

Version(s):
4.14 through to 4.10

Link to docs preview:
* [DNS records](https://64270--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_vsphere/installing-vsphere-installer-provisioned-network-customizations#installation-vsphere-installer-infra-requirements_installing-vsphere-installer-provisioned-network-customizations)

QE review:
- [ ] QE has approved this change.

Additional information:

### CODE REVIEW

File: modules/installation-vsphere-installer-infra-requirements.adoc [hunk 1]
```diff
@@ -464,7 +464,7 @@ endif::vsphere[]
 [discrete]
 [id="installation-vsphere-installer-infra-requirements-dns-records_{context}"]
 === DNS records
-You must create DNS records for two static IP addresses in the appropriate DNS server for the vCenter instance that hosts your {product-title} cluster. In each record, `<cluster_name>` is the cluster name and `<base_domain>` is the cluster base domain that you specify when you install the cluster. A complete DNS record takes the form: `<component>.<cluster_name>.<base_domain>.`.
+You must create DNS records for certain static IP addresses in the appropriate DNS server for the vCenter instance that hosts your {product-title} cluster. In each record, `<cluster_name>` is the cluster name and `<base_domain>` is the cluster base domain that you specify when you install the cluster. A complete DNS record takes the form: `<component>.<cluster_name>.<base_domain>.`.
```
Comment (dfitzmau) [inline modules/installation-vsphere-installer-infra-requirements.adoc:467]: Hello @sgaoshang and @jcpowermac . I swapped a numeric value ("two" or "three") for "certain" so to future proof this entry sentence in the event a DNS record is removed or added.

File: modules/installation-vsphere-installer-infra-requirements.adoc [hunk 2]
```diff
@@ -486,6 +486,12 @@ external to the cluster and from all the nodes within the cluster.
 machines that run the Ingress router pods, which are the worker nodes by
 default. This record must be resolvable by both clients external to the cluster
 and from all the nodes within the cluster.
+
+|Kubernetes API
+|`api-int.<cluster_name>.<base_domain>.`
```
Comment (dfitzmau) [inline modules/installation-vsphere-installer-infra-requirements.adoc:491]: @sgaoshang and @jcpowermac .

Each record name ends with a period (.), this is correct?

**Comment by jcpowermac** [comment]
This is specific to WMCO and Windows. Why would it be added to the vSphere docs?

**Comment by dfitzmau** [comment]
> This is specific to WMCO and Windows. Why would it be added to the vSphere docs?

Thanks, Joe. For minor updates, it's easier to create a PR and see context a proposed change might have on the current docs. I could not see anything in the VMWare docs about this DNS record, so I'll need to reach out to Davide Celano (who reported the bug) about why the vSphere docs were referenced in this way.

**Pull request closed without merging**